src/main/java/seedu/address/model/participation/Participation.java

```diff
@@ -32,6 +32,20 @@ public Participation(Person person, Competition competition, List<Attempt> attem
         this.attempts = attempts;
     }
 
+    /**
+     * Gets the highest score of each of the three lifts in this format: Squat/Bench/Deadlift.
+     * @return a string representation of the three lift score
+     */
+    public String getThreeLiftScore() {
```

Comment:
can you bring this lower and group it with all the other scorring methods in participation